src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -49,7 +49,25 @@ export const View = (props: Props) => {
         </div>
       </header>
 
-      <div className={styles.valuePropositionWrapper}>
+      <section className={styles.quoteWrapper}>
```

Comment:
I believe this is already wrapped in `<main>`, [which is an alternative for `<section>` for the main content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#usage_notes) - so I think a `<div>` for a child of `<main>` was fine.